### PR TITLE
chore: set the default copyright header to something shorter

### PR DESCRIPTION
**Pull request by keturn**
Apache says this is a "shorter variant you may use."

https://www.apache.org/foundation/license-faq.html#Apply-My-Software

this PR makes no attempt to change _all_ the existing copyright headers, but does do a few of the high-level files to set an example.

### CODE REVIEW

File: .idea/copyright/TerasologyEngine_brief.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<component name="CopyrightManager">
+  <copyright>
+    <option name="allowReplaceRegexp" value="20\d\d" />
+    <option name="keyword" value="Copyright.*Moving\s?Blocks" />
```
Comment (skaldarnar) [inline .idea/copyright/TerasologyEngine_brief.xml:4]: Do we want to go for "The Terasology Foundation" instead of "Moving Blocks"? @Cervator ?

Comment (Cervator) [reply .idea/copyright/TerasologyEngine_brief.xml:4]: Yes please - good catch 👍 

That's been a global change far down the todo list for a while now

**Comment by skaldarnar** [comment]
> Even though I know the objective is to make it short I kinda wish there'd be a tiny link in there to explain what "SPDX-License..." refers to. Should we have that explained in a readme somewhere?

I think it's fine to only have the short form in the source files - a developer should be able to type ["SPDX license header"](https://is.gd/nwi4gQ) into their search engine of choice. 
First entry is https://spdx.org/licenses/ to which we _could_ link from the README, if we had a license section ^^ (see launcher has https://github.com/MovingBlocks/TerasologyLauncher#license- - that's where I would put it, if anywhere).

**Comment by keturn** [comment]
We also have `/LICENSE`, and GitHub does recognize that and point its "View License" link there.

**Comment by keturn** [comment]
oh, I missed something by only updating copyright headers in gradle files: Checkstyle validates headers in java files. https://github.com/MovingBlocks/TeraConfig/issues/11

**Pull request merged**